frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -150,6 +178,18 @@
       { taint: false }
     );
   }
+
+  let selectedProject: { name: string, id: string } | undefined = undefined;
```

Comment:
selected project can stay set, even if the related projects list is updated and the project is removed. Meaning you can click 'Ask to join' and it will request a project that's not in the list.

Reply:
I wanted it to stay set so that you can click "No thanks" and then change your mind later, but the case where the related projects list is updated (because you changed the project name or code) is one I didn't think of. Yeah, I'll clear it when the projects list is updated.

Done in commit 8866462e, where I found a way to keep the project selected if it stayed in the list, and clear it if it went away.